SimPEG/PF/Magnetics.py

```diff
@@ -256,6 +224,13 @@ def Intrgl_Fwr_Op(self, m=None, Magnetization="ind"):
 
         return fwr_out
 
+    @property
+    def modelMap(self):
```

Comment:
@fourndo: this is necessary for directives? if so, we can try to address this by providing an `inversion_models` list in the new simulation class so that the inversion is aware of what it is inverting for. (no need to change anything at the moment, just looking forward)

Reply:
ah, I see: also in the Problem. kk, I think we can clean this up a bit in the future